lib/timescaledb/models/model.ex

```diff
@@ -92,4 +92,8 @@ defmodule Membrane.Telemetry.TimescaleDB.Model do
   def add_link(link) do
     Link.changeset(%Link{}, link) |> Repo.insert()
   end
+
+  def add_element_event(element) do
+    Element.changeset(%Element{}, element) |> Repo.insert()
```

Comment:
There is pipe operator and putting argument directly into the function. Can we add one more pipe?
```suggestion
  %Element{}
  |> Element.changeset(element)
  |> Repo.insert()
```
```